### PR TITLE
1.17.0 (release)

### PR DESCRIPTION
**Pull request by nilslice**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,109 @@
...
+  - **fix: clarify error messages around durable objects beta - [nilslice], [pull/1921]**
+
+    Disambiguates error message described in #1859, adds more clarity around other places where Durable Object usage in beta may conflict with `wrangler` functionality.
+
+    [nilslice]: https://github.com/nilslice
+    [pull/1921]: https://github.com/cloudflare/wrangler/pull/1921
+
+  - **Fix filtering by extension for js assets - [rubenamorim], [pull/1722]**
+
+    Fixes #1719
```
Comment (caass) [inline CHANGELOG.md:25]: could you make this a link to the issue 👀

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,109 @@
...
+
+    Update [tokio ecosystem](https://github.com/tokio-rs/tokio) crates including hyper, rustls, openssl and necessary API changes
+    Rewrite get_tunnel_url to use tokio-retry
+
+    [nataliescottdavidson]: https://github.com/nataliescottdavidson
+    [pull/1886]: https://github.com/cloudflare/wrangler/pull/1886
+
+  - **failure --> anyhow - [caass], [pull/1881]**
+
+    Follow up to #1880
```
Comment (caass) [inline CHANGELOG.md:56]: and this one would be cool too

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,109 @@
...
+    has all the same information in an easier to understand and modify
+    format (types).
+
+    [obsidianminor]: https://github.com/ObsidianMinor
+    [pull/1932]: https://github.com/cloudflare/wrangler/pull/1932
+
+  - **sorry i got excited with dependabot - [caass], [pull/1917]**
+
+    [caass]: https://github.com/caass
+    [pull/1917]: https://github.com/cloudflare/wrangler/pull/1917
```
Comment (caass) [inline CHANGELOG.md:87]: this can be...omitted...if you like...

File: Cargo.toml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "wrangler"
-version = "1.16.1"
-authors = ["The Wrangler Team <wrangler@cloudflare.com>", "Avery Harnish <averyharnish@gmail.com>", "Ashley Lewis <ashleymichal@gmail.com>", "Ashley Williams <ashley666ashley@gmail.com>", "Nat Davidson <davidson@cloudflare.com>", "Steve Manuel <nilslice@gmail.com>"]
+version = "1.17.0"
+authors = ["The Wrangler Team <wrangler@cloudflare.com>"]
```
Comment (caass) [inline Cargo.toml:4]: swag 😎

**Review by caass — APPROVED** [review]
cool! some minor nits, but otherwise very excited for this!

File: CHANGELOG.md (hunk 3 above)
Comment (nilslice) [reply CHANGELOG.md:87]: but history! if _you_ like, I will omit. lmk 👍

File: CHANGELOG.md (hunk 1 above)
Comment (nilslice) [reply CHANGELOG.md:25]: ah, yes - why does Github convert this syntax everywhere(?) else into links... but not here. good catch!

File: CHANGELOG.md (hunk 3 above)
Comment (caass) [reply CHANGELOG.md:87]: haha it's fine, thank you though 😆

**Review by Electroid — APPROVED** [review]
🚀

**Pull request merged**